quad_utils/include/quad_utils/quad_kd.h

```diff
@@ -121,6 +122,32 @@ class QuadKD {
                                Eigen::Vector3d joint_state,
                                Eigen::Vector3d &foot_pos_world) const;
 
+  /**
+   * @brief Compute forward kinematics for a specified leg
+   * @param[in] leg_index Spirit leg (0 = FL, 1 = BL, 2 = FR, 3 = BR)
+   * @param[in] body_pos Position of center of body frame
+   * @param[in] body_rpy Orientation of body frame in roll, pitch, yaw
+   * @param[in] joint_state Joint states for the specified leg (abad, hip, knee)
+   * @param[out] g_world_knee Transform of the specified knee in world frame
+   */
+  void worldToKneeFKWorldFrame(int leg_index, Eigen::Vector3d body_pos,
```

Comment:
Eigens missing const &

Reply:
None of these functions are, there's an active Github issue to refactor all of them.